mypy/semanal.py

```diff
@@ -808,6 +806,16 @@ def analyze_metaclass(self, defn: ClassDef) -> None:
     def object_type(self) -> Instance:
         return self.named_type('__builtins__.object')
 
+    def class_subtype(self, info: TypeInfo) -> Type:
```

Comment:
Sounds reasonable.

On Wed, Mar 23, 2016 at 2:38 PM, Guido van Rossum <notifications@github.com>
wrote:

> In mypy/semanal.py
> <https://github.com/python/mypy/pull/1313#discussion_r57169729>:
>
> > @@ -808,6 +806,16 @@ def analyze_metaclass(self, defn: ClassDef) -> None:
> >      def object_type(self) -> Instance:
> >          return self.named_type('__builtins__.object')
> >
> > +    def class_subtype(self, info: TypeInfo) -> Type:
>
> First thing that came to mind. I'll rename to class_type.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/python/mypy/pull/1313/files/f4cf5f4f33e22d9c626f5f04ea6194e71cb17077#r57169729>
>